nupic/research/frameworks/continuous_learning/multihead/subdataset.py

```diff
@@ -36,7 +36,7 @@
 from torchvision.datasets import MNIST
 
 
```

Comment:
How is this class different from [split_dataset](https://github.com/numenta/nupic.research/blob/master/nupic/research/frameworks/pytorch/dataset_utils/dataset_utils.py#L168) defined in `dataset_utils.py`? It looks like this file could be replaced by a few functions in dataset_utils.py

Reply:
[split_dataset](https://github.com/numenta/nupic.research/blob/master/nupic/research/frameworks/pytorch/dataset_utils/dataset_utils.py#L168) doesn't allow for applying a target_transform, so I need to figure out how to be able to transform target variables using just dataset_utils.py

Reply:
I think I resolved the issue, but need to make a slight modification to [split_dataset](https://github.com/numenta/nupic.research/blob/master/nupic/research/frameworks/pytorch/dataset_utils/dataset_utils.py#L168) to include a target_transform